diambra/arena/stable_baselines3/make_sb3_env.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+import diambra.arena
+
+from stable_baselines3.common.vec_env import DummyVecEnv, SubprocVecEnv
+from stable_baselines3.common.utils import set_random_seed
+from stable_baselines3.common.monitor import Monitor
+
+# Make Stable Baselines Env function
+def make_sb3_env(game_id, env_settings, wrappers_settings=None,
+                 use_subprocess=True, seed=0,
+                 start_index=0, allow_early_resets=True,
+                 start_method=None, no_vec=False):
+    """
+    Create a wrapped, monitored VecEnv.
+    :param game_id: (str) the game environment ID
+    :param env_settings: (dict) parameters for DIAMBRA Arena environment
+    :param wrappers_settings: (dict) parameters for environment
+                              wraping function
+    :param start_index: (int) start rank index
+    :param allow_early_resets: (bool) allows early reset of the environment
+    :param start_method: (str) method used to start the subprocesses.
+                        See SubprocVecEnv doc for more information
+    :param use_subprocess: (bool) Whether to use `SubprocVecEnv` or
+                          `DummyVecEnv` when
+    :param no_vec: (bool) Whether to avoid usage of Vectorized Env or not.
+                   Default: False
+    :param seed: (int) initial seed for RNG
+    :return: (VecEnv) The diambra environment
+    """
+
+    env_addresses = os.getenv("DIAMBRA_ENVS", "").split()
+    if len(env_addresses) == 0:
+        raise Exception("ERROR: Running script without DIAMBRA CLI.")
+        sys.exit(1)
+
+    num_envs = len(env_addresses)
+
+    def _make_sb3_env(rank):
+        def _init():
+            env = diambra.arena.make(game_id, env_settings, wrappers_settings,
+                                     seed=seed + rank, rank=rank)
+
+            # Create log dir
+            log_dir = os.path.join("/tmp/DIAMBRALog/", str(rank))
```

Comment:
Wouldn't it make sense to let users configure this?
If it's really just temporarily, we should use https://docs.python.org/3/library/tempfile.html and make sure it gets deleted.

Reply:
I'll update this so that the user can pass this value as argument with this folder as default